openstack.py

```diff
@@ -135,8 +143,8 @@ def nova_list(self, mess, args):
         pt.align = 'l'
 
         for vm in vms:
-            for key, val in vm.networks.items():
-                network = '{}: {}'.format(key, ', '.join(val))
+            for key, vals in vm.networks.items():
+                network = '{}: {}'.format(key, ', '.join(vals))
```

Comment:
I actually don't understand how this works.

I think it has to build up the csv of networks before it puts it into the table under a single column for the current row (`vm`), but it looks like the value of `network` will simply be replaced each time through the `key, values`...